tests/units/feature_extraction/test_feature_calculations.py

```diff
@@ -473,6 +473,47 @@ def test_fft_coefficient(self):
         six.assertCountEqual(self, list(res.index), expected_index)
         self.assertIsNaN(res['coeff_10__attr_"real"'])
 
+    def test_fft_aggregated(self):
+        param = [
+            {"aggtype": "centroid"},
+            {"aggtype": "variance"},
+            {"aggtype": "skew"},
+            {"aggtype": "kurtosis"}
+        ]
+        expected_index = ['aggtype_centroid', 'aggtype_variance', 'aggtype_skew', 'aggtype_kurtosis']
+
+        x = range(10)
+        res = pd.Series(dict(fft_aggregated(x, param)))
+        six.assertCountEqual(self, list(res.index), expected_index)
+        self.assertAlmostEqual(res['aggtype_centroid'], 1.135, places=3)
```

Comment:
This looks like you ran the method on the input and fixated the result (in this case `1.135`).

So, this unit test does not check if the calculation is right. Instead it checks if the value of the feature is not changing between tsfresh version.

I would prefer a result that is calculated by hand. Can you think of a border case, where we can calculate the feature by hand?